app/components/market/submit/parameter-input.component.ts

```diff
@@ -45,7 +52,14 @@ export class ParameterInputComponent implements ControlValueAccessor, OnDestroy
     }
 
     public validate() {
-        return null;
+       const valid = this.parameterValue.valid;
+       if (valid) { return null; }
+       return {
+            validJsonConfig: {
+                valid: false,
+                message: "FormControl is invalid",
```

Comment:
Change the message here to contain what the error is(Look into the parameterValue.errors)

Reply:
done